packages/storefront-lwc/src/modules/commerce/cart/cart.html

```diff
@@ -0,0 +1,204 @@
+<!--
+    Copyright (c) 2020, salesforce.com, inc.
+    All rights reserved.
+    SPDX-License-Identifier: BSD-3-Clause
+    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+-->
+<template>
+    <div role="main" id="maincontent">
+
+
+        <div class="cart-error-messaging cart-error">
+        </div>
+
+        <div class="container">
+            <h1 class="page-title">Your Cart</h1>
+            <div class="row cart-header">
+                <div class="col-sm-4 hidden-xs-down">
+                    <a class="continue-shopping-link" href="/s/RefArch/home?lang=en_US" title="Continue Shopping">
+                        Continue Shopping
+                    </a>
+                </div>
+                <div class="col-sm-3 text-center">
+                    <h2 class="number-of-items">
+                        {products.length} Items
+                    </h2>
+                </div>
+                <div class="col-sm-5 text-right hidden-xs-down">
+                    <div>
+                        <span>Need Help? Call</span>
+                        <span><a class="help-phone-number" href="tel:1-800-555-0199">1-800-555-0199</a></span>
+                    </div>
+                </div>
+            </div>
+            <hr class="no-margin-top">
+        </div>
+
+        <div class="container cart cart-page">
+            <div class="row">
+                <div class="col-md-8">
+                    <template for:each={products} for:item="product" for:index="index">
+                        <div key={product.productId} class="card product-info  uuid-1ef92777cbf6c7daaaafdd1905">
```

Comment:
We don't need the `uuid-1ef92777cbf6c7daaaafdd1905`

Reply:
Cleaned